### PR TITLE
[FLINK-18395][FLINK-18388][docs-zh] Translate "ORC Format" and "CSV Format" page into Chinese

### PR DESCRIPTION
**Pull request by kakaroto928**
## What is the purpose of the change

*(For example: This pull request makes task deployment go through the blob server, rather than through RPC. That way we avoid re-transferring them on each deployment (during recovery).)*

## Brief change log

*(for example:)*
  - *The TaskInfo is stored in the blob store on job creation time as a persistent artifact*
  - *Deployments RPC transmits only the blob storage reference*
  - *TaskManagers retrieve the TaskInfo from the blob cache*

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by kakaroto928** [comment]
@flinkbot thanks for your help, if u have questions about my translation ,don't hesitate to tell me , thanks a lot !

File: docs/dev/table/connectors/formats/csv.zh.md [hunk 1]
```diff
@@ -67,116 +69,112 @@ CREATE TABLE user_behavior (
...
     </tr>
     <tr>
       <td><h5>csv.allow-comments</h5></td>
-      <td>optional</td>
+      <td>可选</td>
       <td style="word-wrap: break-word;">false</td>
       <td>Boolean</td>
-      <td>Ignore comment lines that start with <code>'#'</code> (disabled by default).
-      If enabled, make sure to also ignore parse errors to allow empty rows.</td>
+      <td>对注释的行使用 <code>'#'</code> (默认关闭).如果开启, 忽略解析错误来允许空行的存在.
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/csv.zh.md:131]:
```suggestion
      <td>是否允许忽略注释行（默认不允许），注释行以 <code>'#'</code> 作为起始字符。
      如果允许注释行，请确保 <code>csv.ignore-parse-errors</code> 也开启了从而允许空行。 
```

File: docs/dev/table/connectors/formats/csv.zh.md [hunk 2]
```diff
@@ -67,116 +69,112 @@ CREATE TABLE user_behavior (
...
     </tr>
     <tr>
       <td><h5>csv.ignore-parse-errors</h5></td>
-      <td>optional</td>
+      <td>可选</td>
       <td style="word-wrap: break-word;">false</td>
       <td>Boolean</td>
-      <td>Skip fields and rows with parse errors instead of failing.
-      Fields are set to null in case of errors.</td>
+      <td>对于解析失败的字段和行直接忽略.解析失败的字段直接设置为null.</td>
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/csv.zh.md:139]:
```suggestion
    <td>当解析异常时，是跳过当前字段或行，还是抛出错误失败（默认为 false，即抛出错误失败）。如果忽略字段的解析异常，则会将该字段值设置为<code>null</code>。</td>
```

File: docs/dev/table/connectors/formats/csv.zh.md [hunk 3]
```diff
@@ -67,116 +69,112 @@ CREATE TABLE user_behavior (
...
     </tr>
     <tr>
       <td><h5>csv.array-element-delimiter</h5></td>
-      <td>optional</td>
+      <td>可选</td>
       <td style="word-wrap: break-word;"><code>;</code></td>
       <td>String</td>
-      <td>Array element delimiter string for separating
-      array and row element values (<code>';'</code> by default).</td>
+      <td>分隔数组和行元素的字符串(默认<code>';'</code>).</td>
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/csv.zh.md:146]: ARRAY 和 ROW 类型元素的分隔符

File: docs/dev/table/connectors/formats/csv.zh.md [hunk 4]
```diff
@@ -67,116 +69,112 @@ CREATE TABLE user_behavior (
...
+      <td>转义字符(默认关闭).</td>
     </tr>
     <tr>
       <td><h5>csv.null-literal</h5></td>
-      <td>optional</td>
+      <td>可选</td>
       <td style="word-wrap: break-word;">(none)</td>
       <td>String</td>
-      <td>Null literal string that is interpreted as a null value (disabled by default).</td>
+      <td>null会被转化为空值 (默认关闭).</td>
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/csv.zh.md:160]:
```suggestion
      <td>是否讲 "null" 字符串转化为 null 值。</td>
```

File: docs/dev/table/connectors/formats/csv.zh.md [hunk 5]
```diff
@@ -67,116 +69,112 @@ CREATE TABLE user_behavior (
...
 </table>
 
-Data Type Mapping
+数据类型映射
 ----------------
 
-Currently, the CSV schema is always derived from table schema. Explicitly defining an CSV schema is not supported yet.
-
-Flink CSV format uses [jackson databind API](https://github.com/FasterXML/jackson-databind) to parse and generate CSV string.
+目前CSV框架默认基于表格的形式。 显式的定义一个CSV框架还不被支持。
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/csv.zh.md:168]:
```suggestion
目前 CSV 的 schema 都是从 table schema 推断而来的。显式地定义 CSV schema 暂不支持。
```

File: docs/dev/table/connectors/formats/csv.zh.md [hunk 6]
```diff
@@ -67,116 +69,112 @@ CREATE TABLE user_behavior (
...
+Flink的CSV格式数据使用 [jackson databind API](https://github.com/FasterXML/jackson-databind) 去解析csv字符串。
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/csv.zh.md:169]:
```suggestion
Flink的 CSV Format 数据使用 [jackson databind API](https://github.com/FasterXML/jackson-databind) 去解析 CSV 字符串。
```

File: docs/dev/table/connectors/formats/orc.zh.md [hunk 7]
```diff
@@ -1,8 +1,8 @@
 ---
-title: "Orc Format"
-nav-title: Orc
-nav-parent_id: sql-formats
-nav-pos: 6
+标题: "Orc 格式"
+源标题: Orc
+父id: sql-formats
+位置: 6
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/orc.zh.md:5]: Do not translate.

File: docs/dev/table/connectors/formats/orc.zh.md [hunk 8]
```diff
@@ -23,28 +23,28 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-<span class="label label-info">Format: Serialization Schema</span>
-<span class="label label-info">Format: Deserialization Schema</span>
+<span class="label label-info">格式: 序列化结构</span>
+<span class="label label-info">格式: 反序列化结构</span>
 
-* This will be replaced by the TOC
+* 这里将会被TOC替换
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/orc.zh.md:29]: Do not translate.

File: docs/dev/table/connectors/formats/orc.zh.md [hunk 9]
```diff
@@ -65,17 +65,17 @@ CREATE TABLE user_behavior (
...
 <table class="table table-bordered">
     <thead>
       <tr>
-        <th class="text-left" style="width: 25%">Option</th>
-        <th class="text-center" style="width: 8%">Required</th>
-        <th class="text-center" style="width: 7%">Default</th>
-        <th class="text-center" style="width: 10%">Type</th>
-        <th class="text-center" style="width: 50%">Description</th>
+        <th class="text-left" style="width: 25%">选项</th>
+        <th class="text-center" style="width: 8%">要求</th>
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/orc.zh.md:75]: 是否必选

File: docs/dev/table/connectors/formats/orc.zh.md [hunk 10]
```diff
@@ -65,17 +65,17 @@ CREATE TABLE user_behavior (
...
+        <th class="text-center" style="width: 7%">默认</th>
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/orc.zh.md:76]: 默认值

File: docs/dev/table/connectors/formats/csv.zh.md [hunk 11]
```diff
@@ -23,27 +23,29 @@ specific language governing permissions and limitations
...
-<span class="label label-info">Format: Deserialization Schema</span>
+<span class="label label-info">格式: 序列化结构</span>
+<span class="label label-info">格式: 反序列化结构</span>
 
-* This will be replaced by the TOC
+* 此处被TOC替换
 {:toc}
 
-The [CSV](https://en.wikipedia.org/wiki/Comma-separated_values) format allows to read and write CSV data based on an CSV schema. Currently, the CSV schema is derived from table schema.
+[CSV](https://zh.wikipedia.org/wiki/%E9%80%97%E5%8F%B7%E5%88%86%E9%9A%94%E5%80%BC) 格式允许我们基于CSV框架进行进行数据读写，目前CSV框架是基于表的格式建立的。
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/csv.zh.md:32]:
```suggestion
[CSV](https://zh.wikipedia.org/wiki/%E9%80%97%E5%8F%B7%E5%88%86%E9%9A%94%E5%80%BC) 格式允许我们基于 CSV schema 进行解析和生成 CSV 数据。目前 CSV schema 是基于 table schema 推断而来的。
```

File: docs/dev/table/connectors/formats/csv.zh.md [hunk 12]
```diff
@@ -23,27 +23,29 @@ specific language governing permissions and limitations
...
 | :----------------- | :----------------------|
 | `flink-csv`        | Built-in               |
 
-How to create a table with CSV format
+如何创建CSV格式的表格
 ----------------
 
-Here is an example to create a table using Kafka connector and CSV format.
+
+以下是一个使用kafka connector和CSV格式创建表格的示例。
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/csv.zh.md:48]:
```suggestion
以下是一个使用 Kafka 连接器和 CSV 格式创建表的示例。
```

File: docs/dev/table/connectors/formats/csv.zh.md [hunk 13]
```diff
@@ -23,27 +23,29 @@ specific language governing permissions and limitations
...
 
-| Maven dependency   | SQL Client JAR         |
+为了建立CSV格式，下列的表格提供了为项目使用自动化工具（例如Maven或者SBT）以及SQL客户端使用SQL JAR包的依赖信息。
+
+| Maven依赖           | SQL 客户端 JAR包        |
 | :----------------- | :----------------------|
 | `flink-csv`        | Built-in               |
 
-How to create a table with CSV format
+如何创建CSV格式的表格
```
Comment (wuchong) [inline docs/dev/table/connectors/formats/csv.zh.md:44]:
```suggestion
如何创建使用 CSV 格式的表
```

**Comment by kakaroto928** [comment]
thanks a lot Jark Wu. I will modify these contents asap!

**Comment by kakaroto928** [comment]
hi , Jark Wu
i have just finished modifications in my branch. and now if i should pull a new request or u can already merge those changes ? 
thanks a lot
Weibo PENG

**Comment by kakaroto928** [comment]
Hello Jark Wu
I haven't heard about your advice  since 4 days . i want to know if  everything  is ok or i have to still change something about corrections? 

Thanks for your reply
Weibo PENG

**Comment by wuchong** [comment]
Sorry for the late repley. I just have a 3-days hackthon before. Will review this later today.

**Comment by kakaroto928** [comment]
It's really a great experience for me ,thanks a lot, apprieciate your help!

**Pull request closed without merging**